disnake/types/widget.py

```diff
@@ -36,9 +35,17 @@ class WidgetChannel(TypedDict):
     position: int
 
 
+class WidgetActivity(TypedDict):
+    name: str
+
+
 class WidgetMember(User, total=False):
     nick: str
```

Comment:
do we still receive this? I don't think we do.

Reply:
Good catch, removed it in https://github.com/DisnakeDev/disnake/pull/736/commits/9588b3f51628498009e4966130d4aeeafe8bc66e. That makes this PR a breaking change, but if the API doesn't provide the field anymore, then so be it.
Seems like `name` in the response is now always the nickname (if any), or the username if no nickname is set.